src/libraries/Common/tests/System/Net/Http/Http3LoopbackConnection.cs

```diff
@@ -253,17 +253,6 @@ public async Task ShutdownAsync()
                 // So nothing to do.
                 return;
             }
-            catch (OperationCanceledException)
```

Comment:
I believe we should bring this back? #58078 is not fixed yet

Reply:
I'll bring it back, but interestingly I haven't seen it in any of my local runs nor pipelines...

Reply:
It's a race that is not so easy to catch I guess 😅 but it definitely exists